io_mesh_w3d/import_utils_w3d.py

```diff
@@ -102,20 +101,30 @@ def create_mesh(self, mesh_struct, hierarchy, coll):
         mesh.materials.append(material)
         principleds.append(principled)
 
-    for vertMat in mesh_struct.vert_materials:
+    vert_materials = mesh_struct.vert_materials
+    material_passes = mesh_struct.material_passes
+    textures = mesh_struct.textures
+
+    # When this mesh is using prelit materials use that version
+    if mesh_struct.has_prelit_vertex():
```

Comment:
In case the mesh has prelit vertex materials we use those instead of the regular one (which doesn't exist)